Sources/Service/Environment/Environment+DotEnv.swift

```diff
@@ -0,0 +1,75 @@
+
+import Core
+#if os(Linux)
+    import Glibc
+#else
+    import Darwin
+#endif
+
+extension Environment {
+    /// Loads the ".env" and the ".env.environment" files if they exist
+    /// Both a generic .env file and an environment specific file are loaded.
+    /// This allows you to set custom credentials to specific environments such as
+    /// production, testing, and development.
+    /// If you use custom environments, those will also be loaded.
```

Comment:
Maybe add something explaining that the environment specific ones override the general one.